include/zen/view.h

```diff
@@ -34,6 +34,10 @@ struct zn_view {
   struct zn_board *board;  // nullable
   double x, y;
 
+  // controlled by `zn_view_update_z_index`, called from zn_board
+  // 1 is the backmost
+  unsigned int z_index;
```

Comment:
As a current convention, we prefer explicit bit length.

```suggestion
  uint32_t z_index;
```